libcnb-data/src/newtypes.rs

```diff
@@ -53,10 +53,18 @@ macro_rules! libcnb_newtype {
         $error_name:ident,
         $regex:expr
     ) => {
-        #[derive(Debug, Eq, PartialEq, ::serde::Deserialize, ::serde::Serialize, Clone)]
+        #[derive(Debug, Eq, PartialEq, ::serde::Serialize, Clone)]
         $(#[$type_attributes])*
         pub struct $name(String);
 
+        impl<'de> ::serde::Deserialize<'de> for $name {
```

Comment:
The implementations for `$name` are interleaved between those for `$error_name`. Is this intentional?

Reply:
Nope. :) I'll move the `Deserialize` implementation.